IoTCoreDefaultApp/IoTCoreDefaultApp/IoTCoreDefaultApp.csproj

```diff
@@ -257,6 +253,11 @@
       <Generator>MSBuild:Compile</Generator>
     </Page>
   </ItemGroup>
+  <ItemGroup>
+    <SDKReference Include="WindowsIoT, Version=10.0.14993.0">
+      <Name>Windows IoT Extensions for the UWP</Name>
+    </SDKReference>
+  </ItemGroup>
```

Comment:
Should use Conditional include - only if 14993 is installed. Otherwise, fallback to existing SDK.